### PR TITLE
Add tags for GT components

### PR DESCRIPTION
**Pull request by JuiceyBeans**
## What
Adds tags for each of the GT components, with the intent to make it easier to search for them in XEI

### CODE REVIEW

File: src/generated/resources/data/gtceu/tags/items/sensors.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_sensor",
+    "gtceu:iv_sensor",
+    "gtceu:luv_sensor",
+    "gtceu:zpm_sensor",
+    "gtceu:uv_sensor",
+    "gtceu:uhv_sensor",
+    "gtceu:uev_sensor",
+    "gtceu:uiv_sensor",
+    "gtceu:uxv_sensor",
+    "gtceu:opv_sensor"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/sensors.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/robot_arms.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_robot_arm",
+    "gtceu:iv_robot_arm",
+    "gtceu:luv_robot_arm",
+    "gtceu:zpm_robot_arm",
+    "gtceu:uv_robot_arm",
+    "gtceu:uhv_robot_arm",
+    "gtceu:uev_robot_arm",
+    "gtceu:uiv_robot_arm",
+    "gtceu:uxv_robot_arm",
+    "gtceu:opv_robot_arm"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/robot_arms.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/fluid_regulators.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_fluid_regulator",
+    "gtceu:iv_fluid_regulator",
+    "gtceu:luv_fluid_regulator",
+    "gtceu:zpm_fluid_regulator",
+    "gtceu:uv_fluid_regulator",
+    "gtceu:uhv_fluid_regulator",
+    "gtceu:uev_fluid_regulator",
+    "gtceu:uiv_fluid_regulator",
+    "gtceu:uxv_fluid_regulator",
+    "gtceu:opv_fluid_regulator"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/fluid_regulators.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/field_generators.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_field_generator",
+    "gtceu:iv_field_generator",
+    "gtceu:luv_field_generator",
+    "gtceu:zpm_field_generator",
+    "gtceu:uv_field_generator",
+    "gtceu:uhv_field_generator",
+    "gtceu:uev_field_generator",
+    "gtceu:uiv_field_generator",
+    "gtceu:uxv_field_generator",
+    "gtceu:opv_field_generator"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/field_generators.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/emitters.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_emitter",
+    "gtceu:iv_emitter",
+    "gtceu:luv_emitter",
+    "gtceu:zpm_emitter",
+    "gtceu:uv_emitter",
+    "gtceu:uhv_emitter",
+    "gtceu:uev_emitter",
+    "gtceu:uiv_emitter",
+    "gtceu:uxv_emitter",
+    "gtceu:opv_emitter"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/emitters.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/electric_pumps.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_electric_pump",
+    "gtceu:iv_electric_pump",
+    "gtceu:luv_electric_pump",
+    "gtceu:zpm_electric_pump",
+    "gtceu:uv_electric_pump",
+    "gtceu:uhv_electric_pump",
+    "gtceu:uev_electric_pump",
+    "gtceu:uiv_electric_pump",
+    "gtceu:uxv_electric_pump",
+    "gtceu:opv_electric_pump"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/electric_pumps.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/electric_pistons.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_electric_piston",
+    "gtceu:iv_electric_piston",
+    "gtceu:luv_electric_piston",
+    "gtceu:zpm_electric_piston",
+    "gtceu:uv_electric_piston",
+    "gtceu:uhv_electric_piston",
+    "gtceu:uev_electric_piston",
+    "gtceu:uiv_electric_piston",
+    "gtceu:uxv_electric_piston",
+    "gtceu:opv_electric_piston"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/electric_pistons.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/electric_motors.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_electric_motor",
+    "gtceu:iv_electric_motor",
+    "gtceu:luv_electric_motor",
+    "gtceu:zpm_electric_motor",
+    "gtceu:uv_electric_motor",
+    "gtceu:uhv_electric_motor",
+    "gtceu:uev_electric_motor",
+    "gtceu:uiv_electric_motor",
+    "gtceu:uxv_electric_motor",
+    "gtceu:opv_electric_motor"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/electric_motors.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

File: src/generated/resources/data/gtceu/tags/items/conveyor_modules.json [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    "gtceu:ev_conveyor_module",
+    "gtceu:iv_conveyor_module",
+    "gtceu:luv_conveyor_module",
+    "gtceu:zpm_conveyor_module",
+    "gtceu:uv_conveyor_module",
+    "gtceu:uhv_conveyor_module",
+    "gtceu:uev_conveyor_module",
+    "gtceu:uiv_conveyor_module",
+    "gtceu:uxv_conveyor_module",
+    "gtceu:opv_conveyor_module"
```
Comment (screret) [inline src/generated/resources/data/gtceu/tags/items/conveyor_modules.json]: the high-tier ones need to be marked as optional tag entries so the game won't error.

**Review by screret — CHANGES_REQUESTED** [review]
spotless

**Review by krossgg — APPROVED** [review]

**Review by screret — APPROVED** [review]

**Pull request merged**